src/Endpoint.ts

```diff
@@ -182,5 +142,14 @@ export default class Endpoint {
             console.error("Schema fetch error: " + e.message);
             throw e;
         }
+
+        if (this.timeOut) {
+            clearTimeout(this.timeOut);
+            this.timeOut = null;
+        }
+
+        // somehow TS 2.8.3 doesn't like this.timeOut = setTimeout(this.updateEndpoints.bind(this), this.timeOutDuration);
+        // (typeof this.timeout == Timer | null vs typeof setTimeout() == number...)
```

Comment:
I think I had the same issue, but it was solved by adding
`import { setTimeout } from "timers";`

Reply:
Thank you fixed with 527b5249fe577fab791246ce5a42a9fa8a212b49